libevmasm/Assembly.h

```diff
@@ -149,6 +150,7 @@ class Assembly
 	std::vector<size_t> decodeSubPath(size_t _subObjectId) const;
 	size_t encodeSubPath(std::vector<size_t> const& _subPath);
 
+	Assembly(std::string _name = ""):m_name(std::move(_name)) { }
```

Comment:
Constructors are usually at the top. Also please avoid the conversion from `char *` to `string`